docker-compose-sgx.yaml

```diff
@@ -16,32 +16,48 @@ version: '3.5'
 
 services:
   tcf:
```

Comment:
change the service name to "avalon-shell"

Reply:
Ok